<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>283</x>
      <y>10</y>
      <width>20</width>
      <height>381</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="makeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Составить
матрицу</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-Deikstra-Desktop_Qt_5_0_2_MinGW_32bit-Debug/102New.ICO</normaloff>../build-Deikstra-Desktop_Qt_5_0_2_MinGW_32bit-Debug/102New.ICO</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="showDeWay">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>420</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт поиска</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-Deikstra-Desktop_Qt_5_0_2_MinGW_32bit-Debug/102157.ICO</normaloff>../build-Deikstra-Desktop_Qt_5_0_2_MinGW_32bit-Debug/102157.ICO</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>420</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-Deikstra-Desktop_Qt_5_0_2_MinGW_32bit-Debug/Channel Screen Saver0.ico</normaloff>../build-Deikstra-Desktop_Qt_5_0_2_MinGW_32bit-Debug/Channel Screen Saver0.ico</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="vertex">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить вершины</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-Deikstra-Desktop_Qt_5_0_2_MinGW_32bit-Debug/102Point.ico</normaloff>../build-Deikstra-Desktop_Qt_5_0_2_MinGW_32bit-Debug/102Point.ico</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="vertexLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество вершин</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="countSpin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="startLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QLabel" name="targetLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Цель</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="startSpin">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="targetSpin">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="MatrixCreate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Обработать</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="TableCreate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="Graphics" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>341</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="edge">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить ребра    </string>
    </property>
   </widget>
   <widget class="QPushButton" name="getButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить из файла
матрицу смежности</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>320</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить
граф</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-Deikstra-Desktop_Qt_5_0_2_MinGW_32bit-Debug/102Rand.ico</normaloff>../build-Deikstra-Desktop_Qt_5_0_2_MinGW_32bit-Debug/102Rand.ico</iconset>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>671</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="resText">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>440</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="resLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>420</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Результат</string>
    </property>
   </widget>
   <widget class="QLabel" name="actionLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>420</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>&lt;&lt;Действие&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="toEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>440</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="fromEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="weightEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="change_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>320</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить
граф</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-Deikstra-Desktop_Qt_5_0_2_MinGW_32bit-Debug/102Rand.ico</normaloff>../build-Deikstra-Desktop_Qt_5_0_2_MinGW_32bit-Debug/102Rand.ico</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="delayEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>439</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>600</string>
    </property>
   </widget>
   <widget class="QLabel" name="delayLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>419</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Время пауз, ms</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>663</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Загрузить пример</string>
    </property>
    <addaction name="action_1"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_1">
   <property name="text">
    <string>Пример № 1</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Пример № 2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
